Opserver.Core/Settings/RedisSettings.cs

```diff
@@ -50,9 +50,15 @@ public class Instance : ISettingsCollectionItem
             public string Password { get; set; }
 
             /// <summary>
+            /// Specify if ssl should be used. Defaults to false.
+            /// </summary>
+            public bool Ssl { get; set; } = false;
```

Comment:
let's call this `UseSSL` so it's clear that it's a bool and consistent with naming (e.g. "CPU" is all caps in settings)